### PR TITLE
Update sample app READMEs with web instructions

### PR DESCRIPTION
**Pull request by fryn**

### CODE REVIEW

**Comment by fryn** [comment]
@JakeWharton, @swankjesse, for some reason, I cannot edit the reviewers field. Do I need to be granted write/maintainer access for this repository?

File: samples/emoji-search/README.md [hunk 1]
```diff
@@ -7,12 +7,16 @@ This is a mobile app demo of Zipline.
...
 
 Prerequisites
 -------------
 
-In order to build and run these applications you'll need to:
-- Have Android Studio installed
+In order to build and run these applications you'll need to have the following installed:
+- Android Studio
+- gradle
+- git-lfs
```
Comment (adrw) [inline samples/emoji-search/README.md:19]: What do we need git-lfs for?

**Review by adrw — APPROVED** [review]

File: samples/emoji-search/README.md (hunk 1 above)
Comment (fryn) [reply samples/emoji-search/README.md:19]: Honestly I'm not sure, but when I did not have git-lfs installed, `./gradlew samples:counter:browser:browserRun` failed to complete and threw some error that referred to git-lfs. Once I installed it, everything worked.

Comment (fryn) [reply samples/emoji-search/README.md:19]: I don't have write access to this repository, so someone will need to merge this for me.

**Review by swankjesse — APPROVED** [review]

File: samples/counter/README.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+In order to build and run these applications you'll need to have the following installed:
+- gradle
+- git-lfs
+
+
+Running Counter on Web
+----------------------
+
+To simply view the sample app, open: [[https://cashapp.github.io/redwood/latest/counter/]]
```
Comment (JakeWharton) [inline samples/counter/README.md]: I think this was MediaWiki syntax? Markdown will linkify raw links, or you can use `[this](syntax)` for linking plain text.

```suggestion
To simply view the sample app, open: https://cashapp.github.io/redwood/latest/counter/
```

File: samples/counter/README.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+
+To build and modify the app locally, run this:
+```
+./gradlew samples:counter:browser:browserRun
+```
+
+If successful, the command will load the app at [[http://localhost:8080/]] in your default web browser.
```
Comment (JakeWharton) [inline samples/counter/README.md]:
```suggestion
If successful, the command will load the app at http://localhost:8080/ in your default web browser.
```

File: samples/emoji-search/README.md (hunk 1 above)
Comment (JakeWharton) [reply samples/emoji-search/README.md:19]: The paparazzi snapshots are stored in LFS

File: samples/emoji-search/README.md [hunk 4]
```diff
@@ -62,6 +66,21 @@ Then build and run the app. The shared Kotlin code will be built automatically a
 The app pulls the JavaScript from the presenters server and requires it to be running in order to work.
 
 
+Running Emoji-Search on Web
+---------------------------
+
+To simply view the sample app, open: [[https://cashapp.github.io/redwood/latest/emoji-search/]]
```
Comment (JakeWharton) [inline samples/emoji-search/README.md]:
```suggestion
To simply view the sample app, open: https://cashapp.github.io/redwood/latest/emoji-search/
```

File: samples/emoji-search/README.md [hunk 5]
```diff
@@ -62,6 +66,21 @@ Then build and run the app. The shared Kotlin code will be built automatically a
...
+
+To build and modify the app locally, run this:
+```
+./gradlew samples:emoji-search:browser:browserRun
+```
+
+If successful, the command will load the app at [[http://localhost:8080/]] in your default web browser.
```
Comment (JakeWharton) [inline samples/emoji-search/README.md]:
```suggestion
If successful, the command will load the app at http://localhost:8080/ in your default web browser.
```

**Review by JakeWharton — APPROVED** [review]

**Pull request merged**